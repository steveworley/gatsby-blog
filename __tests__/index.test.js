import React from 'react'
import renderer from 'react-test-renderer'
import Index from '../src/pages/index'

import { stub } from '../__mocks__/index.mock'

jest.mock('../src/components/layout',  () => 'Layout')

describe("Index", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Index data={stub} />).toJSON()
    expect(tree).toMatchSnapshot();
  })
})