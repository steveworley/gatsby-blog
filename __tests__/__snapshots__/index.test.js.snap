// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders correctly 1`] = `
<Layout>
  <div
    id="main"
  >
    <section
      id="one"
    >
      <header
        className="major"
      >
        <h2>
          Youtube z-index fix
        </h2>
      </header>
      <p>
        Flash appearing behind elements is a common problem with a relatively simple fix â€” simply add wmode=transparent to the object tag and your Flash movie will behave nicely with the CSS z-index property.
      </p>
      <ul
        className="actions"
      >
        <li>
          <a
            className="button"
            href="youtube-z-index-fix"
          >
            Continue reading
          </a>
        </li>
      </ul>
    </section>
    <section
      id="two"
    >
      <h2>
        Recent Posts
      </h2>
      <ul
        className="actions"
      >
        <li>
          <a
            href="css-attribute-selectors"
          >
            CSS attribute Selectors
          </a>
        </li>
      </ul>
    </section>
  </div>
</Layout>
`;
